doc/rtd/reference/yaml_examples/apt_repos.rst

```diff
@@ -0,0 +1,78 @@
+.. _cce-add-apt-repos:
+
+Add primary APT repositories
```

Comment:
I think we should combine this with the other apt page.

Reply:
Makes sense to me!